parkings/api/enforcement/urls.py

```diff
@@ -1,9 +1,11 @@
 from django.conf.urls import include, url
 from rest_framework.routers import DefaultRouter
 
+from .operator import OperatorViewSet
 from .valid_parking import ValidParkingViewSet
 
 router = DefaultRouter()
+router.register('operator', OperatorViewSet, base_name='operator')
```

Comment:
`base_name` not needed here.

Reply:
I like the explicit base_name more. It's kind of "magic" when its determined from the model behind the serializer's queryset.